### PR TITLE
Add instruction to access UI without IAP

### PR DESCRIPTION
**Pull request by IronPan**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/461)

### CODE REVIEW

File: content/docs/pipelines/pipelines-quickstart.md [hunk 1]
```diff
@@ -25,6 +25,8 @@ Follow these steps to deploy Kubeflow and open the pipelines dashboard:
     alt="Kubeflow UI"
     class="mt-3 mb-3 border border-info rounded">
 
+   If IAP is skipped, run ```kubectl port-forward -n kubeflow `kubectl get pods -n kubeflow --selector=service=ambassador -o jsonpath='{.items[0].metadata.name}'` 8080:80``` and go to http://localhost:8080/
```
Comment (sarahmaddox) [inline content/docs/pipelines/pipelines-quickstart.md]: Please fix indentation so that the numbered list continues properly from the next point onwards.

Comment (sarahmaddox) [inline content/docs/pipelines/pipelines-quickstart.md]: For clarity, change the following:

If IAP is skipped

To this:

If you skipped the IAP option when deploying Kubeflow

**Comment by sarahmaddox** [comment]
Friendly ping @IronPan How's it going with these updates?

**Comment by sarahmaddox** [comment]
Friendly ping @IronPan Would you like to make the updates suggested in the review?

**Comment by IronPan** [comment]
@sarahmaddox Thanks for the suggestions. Updated the doc

**Pull request merged**